stdlib/2/__builtin__.pyi

```diff
@@ -29,6 +29,7 @@ class classmethod: pass  # Special, only valid as a decorator.
 class object:
     __doc__ = ...  # type: Optional[str]
     __class__ = ...  # type: type
+    __slots__ = ...  # type: Optional[Iterable[Union[str, unicode]]]
```

Comment:
Tested with different types of iterables, including generators, this works. However, there is one surprise in the API of __slots__. Consider:
```
>>> class C(object):
...   __slots__ = 'one'
...
>>> c = C()
>>> c.one = 'valid'
>>> c.o = 'invalid'
Traceback (most recent call last):
...
AttributeError: 'C' object has no attribute 'o'
```

Your type hint supports this use case implicitly because a string is also an iterable of strings. However, this suggests to the reader that providing `'one'` should be equivalent to providing `('o', 'n', 'e')` and it's not. I think it would be clearer to list the "str" and "unicode" types explicitly for that very reason.